docs/sources/flow/reference/components/prometheus.exporter.statsd.md

```diff
@@ -0,0 +1,108 @@
+---
+# NOTE(rfratto): the title below has zero-width spaces injected into it to
+# prevent it from overflowing the sidebar on the rendered site. Be careful when
+# modifying this section to retain the spaces.
+#
+# Ideally, in the future, we can fix the overflow issue with css rather than
+# injecting special characters.
+
+title: prometheus.exporter.statsd
+---
+
+# prometheus.exporter.statsd
+The `prometheus.exporter.statsd` component embeds
+[statsd_exporter](https://github.com/prometheus/statsd_exporter) for collecting StatsD-style metrics and exporting them as Prometheus metrics.
+
+## Usage
+
+```river
+prometheus.exporter.statsd "LABEL" {
+    
+}
+```
+
+## Arguments
+The following arguments can be used to configure the exporter's behavior.
+All arguments are optional. Omitted fields take their default values.
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`listen_udp`                                      | `string`       | The UDP address on which to receive statsd metric lines. Use "" to disable it. | `9125` | mo
+`listen_tcp`                                      | `string`       | The UDP address on which to receive statsd metric lines. Use "" to disable it. | `9125` | no
```

Comment:
Looks like this line may be duplicated
```suggestion
`listen_tcp`                                      | `string`       | The UDP address on which to receive statsd metric lines. Use "" to disable it. | `9125` | no
```

Reply:
I don't think so. First is `udp`, second `tcp`. The description is duplicated tho